scr/splitInteger.test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const { splitInteger } = require('./splitInteger');
+// const { parts } = require('./splitInteger');
+
+test(`should split a number into equal parts if a value
+ is divisible by  a numberOfParts`, () => {
+  expect(splitInteger(16, 4)).toEqual([4, 4, 4, 4]);
+});
+
+test(`should return a part equals to a value
+  when splitting into 1 part`, () => {
+  const mas = [4];
+
+  expect(splitInteger(4, 1).length).toEqual(mas.length);
+});
+
+test('should return corect equal', () => {

```

Comment:
Could you split these tests according to the test design technics that you have used for creating these test datasets?
Also, revise tests naming - test titles must reflect the goal of the test.